doc/classes/TabContainer.xml

```diff
@@ -74,7 +78,7 @@
 			<argument index="0" name="tab_idx" type="int">
 			</argument>
 			<description>
-				Returns [code]true[/code] if the tab at index [code]tab_idx[/code] is disabled.
+				Returns [code]true[/code] if the tab at index [code]tab_idx[/code] is disabled. Returns [code]false[/code] otherwise.
```

Comment:
Second sentence is redundant and can be removed.

Reply:
fixed